### PR TITLE
feat: Add cancel order component to deprecated limit orders

### PR DESCRIPTION
**Pull request by memoyil**
---

## PR-Codex overview
This PR introduces the handling of existing limit orders in the application, enhancing the user interface with a new component for displaying these orders. It also implements caching for transaction queries to improve performance.

### Detailed summary
- Added `ORDER_CATEGORY.Existing` to handle existing orders.
- Created `ExistingOrder` interface for existing limit order data structure.
- Introduced `ExistingLimitOrderTable` component to display existing orders.
- Implemented caching mechanism in `handler` for transaction queries.
- Updated `LimitOrderTable` to include logic for rendering existing orders.
- Modified localization files to add translations for existing orders.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/views/LimitOrders/components/LimitOrderTable/ExistingLimitOrderTable.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                        {isMobile ? truncateHash(order[0]) : order[0]}
+                        <BscScanIcon color="invertedContrast" ml="4px" />
+                      </Link>
+                    </Flex>
+                  </Box>
+                </Flex>
+              </Td>
+              <Td>
+                <Button
+                  onClick={async () => {
```
Comment (chef-eric) [inline apps/web/src/views/LimitOrders/components/LimitOrderTable/ExistingLimitOrderTable.tsx:80]: how about wrap a useCallback

File: apps/web/src/views/LimitOrders/components/LimitOrderTable/index.tsx [hunk 2]
```diff
@@ -5,35 +5,31 @@ import useGelatoLimitOrdersHistory from '../../hooks/useGelatoLimitOrdersHistory
...
 const LimitOrderTable: React.FC<React.PropsWithChildren<{ isCompact: boolean }>> = ({ isCompact }) => {
   const { t } = useTranslation()
-  const [activeTab, setIndex] = useState<ORDER_CATEGORY>(ORDER_CATEGORY.Open)
+  const [activeTab, setIndex] = useState<ORDER_CATEGORY>(ORDER_CATEGORY.Existing)
   const handleClick = useCallback((tabType: ORDER_CATEGORY) => setIndex(tabType), [])
   const tabMenuItems = useMemo(() => {
+    if (activeTab === ORDER_CATEGORY.Existing) {
+      return [t('Existing Orders')]
+    }
     return [t('Open Orders'), t('Expired Order'), t('Order History')]
```
Comment (chef-eric) [inline apps/web/src/views/LimitOrders/components/LimitOrderTable/index.tsx:31]: so there will still show the ('Open Orders'), t('Expired Order'), t('Order History') tabs? or just drop them

Comment (chef-eric) [reply apps/web/src/views/LimitOrders/components/LimitOrderTable/index.tsx:31]: when i click the tabMenu header, the activeTab will be 0, so then the tabMenuItems will render to the old three tabs, and i can't back to the 'Existing Orders' tab

**Review by chef-eric — DISMISSED** [review]

**Comment by chef-eric** [comment]
oh, some conflict

**Review by chef-eric — APPROVED** [review]

**Review by chef-ryan — APPROVED** [review]

**Pull request merged**